opm/input/eclipse/EclipseState/Runspec.hpp

```diff
@@ -218,6 +232,9 @@ class NetworkDims {
     int nMaxNoNodes;
     int nMaxNoBranches;
     int nMaxNoBranchesConToNode;
+
+    bool extNetwork_{false};
```

Comment:
Having both of these implies that it is possible to have both extended and standard network at the same time. This class' constructors correctly only sets one or the other, but maybe an enum with values Standard, Extended, None or similar would have been better?

Reply:
> Having both of these implies that it is possible to have both extended and standard network at the same time. This class' constructors correctly only sets one or the other, but maybe an enum with values Standard, Extended, None or similar would have been better?

That's a good idea.  I've pushed an update to that effect.